app/app.module.js

```diff
@@ -148,8 +148,11 @@ $.get('static/conf/plugins.json', function (plugins) {
         $stateProvider, SettingsProvider) {
 
         var stmt = 'SELECT settings[\'license\'][\'enterprise\'] as enterprise, ' +
-          'settings[\'license\'][\'ident\'] as ident ' +
-          'FROM sys.cluster';
+//                    'settings[\'license\'][\'ident\'] as ident ' +
+                        'if(license is not NULL, ' +
+                            'format(\'%s (exp: %tD)\', license[\'issuedTo\'], license[\'expirationDateInMs\']), ' +
```

Comment:
please use `%tF` for ISO date instead of US format that nobody else uses.

Reply:
done